### PR TITLE
Options URL.

### PR DESCRIPTION
**Pull request by hemanth**
Fixed.

### CODE REVIEW

**Comment by sebmck** [comment]
Thanks!

File: docs/usage/browser.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ test.test(); // "test"
 
 Programmatically transpile and execute strings of ES6 code.
 
-See [options](#options) for additional documentation.
+See [options](http://6to5.org/docs/usage/options/) for additional documentation.
```
Comment (thejameskyle) [inline docs/usage/browser.md]: This should be just `/docs/usage/options/`

Comment (hemanth) [inline docs/usage/browser.md]: Missing something ?

Comment (thejameskyle) [inline docs/usage/browser.md]: We shouldn't qualify it with the domain

Comment (hemanth) [inline docs/usage/browser.md]: Ha, ok.

Comment (sebmck) [inline docs/usage/browser.md]: @hemanth Already been fixed.

**Pull request merged**